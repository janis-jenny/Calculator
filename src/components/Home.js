const Home = () => (
  <div>
    <h1>My Calculator</h1>
  </div>
);

export default Home;
