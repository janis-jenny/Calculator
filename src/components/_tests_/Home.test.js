import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

it('Home component renders correctly', () => {
  const tree = renderer.create(<Home />);
  expect(tree).toMatchSnapshot();
});
