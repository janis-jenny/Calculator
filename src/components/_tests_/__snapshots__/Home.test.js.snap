// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
<div
  className="home"
>
  <h1>
    Where Math Magic Happens
  </h1>
  <img
    alt=""
    src="magician.gif"
  />
</div>
`;
